docs/Marketplace.md

```diff
@@ -402,13 +476,13 @@ POSTing a storage request will make it available in the storage market, and a st
 You can poll the status of your request by means of:
 ```bash
 export STORAGE_PURCHASE_ID="1d0ec5261e3364f8b9d1cf70324d70af21a9b5dccba380b24eb68b4762249185"
-curl "http://localhost:8001/api/codex/v1/storage/purchases/${STORAGE_PURCHASE_ID}"
+curl "http://localhost:8001/api/codex/v1/storage/purchases/${STORAGE_PURCHASE_ID}" | jq
```

Comment:
:yellow_circle: This assumes people have `jq` installed, which is not necessarily true. I'd remove it, but won't fight for it or block the PR because of it.

Reply:
True, but if you do not mind I would leave it for a moment. I added a sentence explaining what it is, and how to get it installed quickly on macOS. It makes reading the output so much easier...